src/Commands/MpxImporter.php

```diff
@@ -129,13 +127,16 @@ public function warmKeyValue(string $media_type_id) {
     $list = $request->wait();
 
     // @todo Support fetching the total results via ObjectList.
-    $this->io()->title(dt('Warming key-value for @type media', ['@type' => $media_type_id]));
+    $this->io()->title(dt('Warming cache for @type media', ['@type' => $media_type_id]));
     $this->io()->progressStart();
 
-    $importer = new DataObjectImporter($this->keyValueFactory, $this->entityTypeManager);
-    each_limit($list->yieldLists(), 4, function (ObjectList $list) use ($media_type, $importer, &$count) {
-      foreach ($list as $item) {
-        $importer->setKeyValue($item, $media_type);
+    $service_info = $media_source->getPluginDefinition()['media_mpx'];
+
+    each_limit($list->yieldLists(), 4, function (ObjectList $list) use ($media_type, $service_info) {
+      foreach ($list as $index => $item) {
+        // For each item, we need to inject it into the response cache as if
+        // a single request for the item was made.
+        $this->dataObjectImporter->cacheItem($item, $service_info);
```

Comment:
This is our hack to fill the cache without re-loading each object again from mpx.